nexus/db-queries/src/db/datastore/rack.rs

```diff
@@ -629,6 +646,44 @@ impl DataStore {
                         })?;
                     }
 
+                    // Insert the RSS-generated blueprint.
+                    Self::blueprint_insert_on_connection(
+                        &conn, opctx, &blueprint,
+                    )
+                    .await
+                    .map_err(|e| {
+                        warn!(
+                            log,
+                            "Initializing Rack: Failed to insert blueprint"
```

Comment:
Can we include the error details here?

Reply:
I rolled all these log changes up (including other rack errors that weren't in the diff) into [f1acada](https://github.com/oxidecomputer/omicron/pull/5244/commits/f1acada1266311c7ae518ff570d69c842dcecb01)